test/jdk/jdk/dynalink/TypeConverterFactoryMemoryLeakTest.java

```diff
@@ -74,9 +76,7 @@ public GuardedInvocation convertToType(Class<?> sourceType, Class<?> targetType,
     }
 
     public static void main(String[] args) {
-        long start = System.nanoTime();
-        long deadline = start + MAX_DURATION.toNanos();
-        while (System.nanoTime() < deadline) {
+        for (int count = 0; count < MAX_ITERATIONS; ++count) {
```

Comment:
Here and later: use postfix `count++`, regular style?

Reply:
Old habits die hard; I think I started doing this about 30 years ago when writing C code against a compiler on Atari ST that emitted more efficient MC68000 code for `++i` than for `i++`. I guess it's time to unlearn this :-) Of course, if you wanted to get the fastest code, you would've counted _decrementing to zero_ to let the compiler use the DBRA (decrement and branch) instruction.

It's funny how I _don't_ miss those days.

Reply:
Good!